### PR TITLE
Project structure

### PR DESCRIPTION
**Pull request by el-dub**
Add maven configuration and test controller.

### CODE REVIEW

**Review by rudik-byte — DISMISSED** [review]

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <groupId>org.springframework.boot</groupId>
+        <version>2.4.3</version>
+    </parent>
+    <groupId>com.discount</groupId>
```
Comment (SergeyKoval) [inline pom.xml]: group id should contain information about organization. com.exade as example

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    <artifactId>2021_june_team4_be</artifactId>
```
Comment (SergeyKoval) [inline pom.xml]: artifact id should specify some product name. discount as example

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.20</version>
```
Comment (SergeyKoval) [inline pom.xml]: move version to properties

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.13.2</version>
```
Comment (SergeyKoval) [inline pom.xml]: move version to properties

File: src/main/java/com/discount/Application.java [hunk 5]
```diff
@@ -0,0 +1,12 @@
+package com.discount;
```
Comment (SergeyKoval) [inline src/main/java/com/discount/Application.java]: change package structure based on pom group id and artifact id: com.exadel.discount

**Review by SergeyKoval — CHANGES_REQUESTED** [review]

**Review by rudik-byte — APPROVED** [review]

**Review by rudik-byte — DISMISSED** [review]

File: pom.xml [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <groupId>org.springframework.boot</groupId>
+        <version>2.4.3</version>
+    </parent>
+    <groupId>com.exadel</groupId>
+    <artifactId>Discount</artifactId>
```
Comment (SergeyKoval) [inline pom.xml]:
```suggestion
    <artifactId>discount</artifactId>
```

File: src/main/java/com/exadel/discount/Application.java [hunk 7]
```diff
@@ -0,0 +1,12 @@
+package com.exadel.discount;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class Application {
```
Comment (SergeyKoval) [inline src/main/java/com/exadel/discount/Application.java]:
```suggestion
public class DiscountApplication {
```

**Review by rudik-byte — APPROVED** [review]

**Review by SergeyKoval — APPROVED** [review]

**Pull request merged**